pages/HallOfFame/index.js

```diff
@@ -0,0 +1,98 @@
+import StyledHeading from "@/components/Styles/StyledHeading";
+import StyledLink from "@/components/Styles/StyledLink";
+import useSWR from "swr";
+import { formatPetsAge } from "@/utils";
+import PNGImage from "@/components/PNGImage";
+import styled from "styled-components";
+
+export default function HallOfFame() {
+  const { data, isLoading } = useSWR("/api/pets");
+  if (isLoading) {
+    return <h1>Loading...</h1>;
+  }
+  if (!data) {
+    return;
+  }
+
+  function sortedDeceasedPets(data) {
+    let arrayCopy = [...data];
+    arrayCopy.sort(
+      (a, b) => b.lastUpdated - b.createdAt - (a.lastUpdated - a.createdAt)
+    );
+    return arrayCopy.slice(0, 100);
+  }
+
+  const top100 = sortedDeceasedPets(data);
+  const top100Index = top100.map((item, index) => ({
+    ...item,
+    originalIndex: index,
+  }));
+
+  return (
+    <StyledBackground>
+      <div>
+        <StyledLink href={"/"}> ←</StyledLink>
+        <StyledHeading>Hall of Fame</StyledHeading>
+      </div>
+      <StyledList>
+        {top100Index.map((pet) => (
+          <StyledListItems key={pet._id}>
+            <p></p>
```

Comment:
Did you need that or you want to put something after?

Reply:
you are right we dont need that we will change that later